llama_stack/apis/tools/rag_tool.py

```diff
@@ -45,6 +45,13 @@ class RAGQueryGenerator(Enum):
     custom = "custom"
 
 
+@json_schema_type
+class RAGSearchMode(Enum):
+    VECTOR = "vector"
+    KEYWORD = "keyword"
+    HYBRID = "hybrid"
```

Comment:
nit: We could add docstrings explaining each mode, what do you think?

    """Search modes for RAG query retrieval.
    
    VECTOR: Uses vector similarity search for semantic matching
    KEYWORD: Uses keyword-based search for exact matching
    HYBRID: Combines both vector and keyword search for better results
    """

Reply:
There are some instances of docstrings in this file too.